app/assets/javascripts/show_errors.js

```diff
@@ -17,8 +17,9 @@ window.Champaign.showErrors = function(e, data) {
   response = $.parseJSON(data.responseText);
 
   var errorMsg = function(field_name, msgs) {
-    msg = (typeof msgs === "string") ? msgs : msgs[0]
-    return ["<div class='error-msg'>This field ", msg, "</div>"].join('');
+    var msg = (typeof msgs === "string") ? msgs : msgs[0]
+    var prefix = window.I18n ? I18n.t('errors.this_field') : 'This field';
```

Comment:
yeah - showErrors is used on the member facing and the campaigner facing UIs. I18n is not included on the campaigner facing side because they said they don't have any intention to i14e it in the near future.